src/mixins/filing-template-mixin.ts

```diff
@@ -396,6 +396,10 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
         }
       }
       delete party.officer.businessNumber
+
+      // if tax id is empty (which doesn't match the schema pattern), delete it
+      if (!party.officer.taxId) delete party.officer.taxId
```

Comment:
If the user did not touch the input control for this value (which is labelled Business Number) then this v-model value is null or undefined, which is fine.

But if the user did touch the input control then this v-model value is "" (empty string), which fails schema validation due to the [required pattern].(https://github.com/bcgov/business-schemas/blob/e32e6dc561abebdc3034af48993da12f882dfb80/src/registry_schemas/schemas/party.json#L46).

Therefore, the solution is to delete this property if it's empty (null, undefined or empty string).